src/main/kotlin/no/nav/syfo/aktivitetskrav/domain/AktivitetskravVarsel.kt

```diff
@@ -52,3 +52,7 @@ data class AktivitetskravVarsel internal constructor(
         svarfrist = this.svarfrist,
     )
 }
+
+enum class VarselType {
+    FORHANDSVARSEL_STANS_AV_SYKEPENGER
```

Comment:
Hmm, trenger den `_STANS_AV_SYKEPENGER` suffixen?
Jeg ser bare for meg at en av defremtidige typene kommer til å være bare `STANS_AV_SYKEPENGER`, og da blir det litt rart å ha `FORHANDSVARSEL_STANS_AV_SYKEPENGER` og `STANS_AV_SYKEPENGER`?

Er ikke så farlig, bare en tanke jeg hadde 🤷🏼‍♂️

Reply:
Bra kommentar🙏 Som ny i domenet synes jeg ikke et varsel om et `FORHANDSVARSEL` gir meg så mye. En ny utvikler som hadde kommet inn på teamet gjetter jeg også måtte ha spurt hva et `FORHANDSVARSEL` er, imens et `FORHANDSVARSEL_STANS_AV_SYKEPENGER` er selvforklarende.
Synes ofte/nesten alltid at tydelige variabel navn som ikke kan misforstås er verdt de ekstra bokstavene 😊

Tenker også at i fremtiden når/hvis vi får `STANS_AV_SYKEPENGER` så vil det være tydelig at `FORHANDSVARSEL_STANS_AV_SYKEPENGER` er et forhåndsvarsel for selve varselet om at det nå er stans imens `FORHANDSVARSEL` kunne vært et forhåndsvarsel til hva som helst.

Reply:
Godt poeng av Vetle, det finnes vel andre forhåndsvarsler, selv om vi kanskje ikke skal implementere noen av de 🤔

Reply:
<img src="https://media2.giphy.com/media/LQ0wA2LZFXiINe0UG7/giphy.gif"/>